### PR TITLE
deleted legacy plugins

### PR DESCRIPTION
**Pull request by Nine-H**
ipod, lastfm, cds basically anything that's more at home in 2007 than 2017

### CODE REVIEW

**Comment by mrnhmath** [comment]
There's still people using Last.fm, including me..

**Comment by Bat41** [comment]
@mrnhmath I don't really know Last.fm. What are you (and other people if you know) using it for?

File: plugins/Devices/CMakeLists.txt [hunk 1]
```diff
@@ -1,3 +1 @@
 add_subdirectory (AudioPlayers)
```
Comment (Bat41) [inline plugins/Devices/CMakeLists.txt]: Are we keeping this plugin?

File: .travis.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@ services:
  - docker
 
 env:
- - DEPENDENCY_PACKAGES="cmake libaccounts-glib-dev libclutter-gtk-1.0-dev libdbusmenu-glib-dev libdbusmenu-gtk3-dev libdbus-glib-1-dev libindicate-dev libgda-5.0-dev libgee-0.8-dev libglib2.0-dev libgpod-dev libgranite-dev libgsignon-glib-dev libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libgtk-3-dev libjson-glib-dev libnotify-dev libpeas-dev libsoup2.4-dev libtagc0-dev libwebkitgtk-dev libxml2-dev libzeitgeist-2.0-dev valac"
+ - DEPENDENCY_PACKAGES="cmake libaccounts-glib-dev libclutter-gtk-1.0-dev libdbusmenu-glib-dev libdbusmenu-gtk3-dev libdbus-glib-1-dev libindicate-dev libgda-5.0-dev libgee-0.8-dev libglib2.0-dev libgranite-dev libgsignon-glib-dev libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libgtk-3-dev libjson-glib-dev libnotify-dev libpeas-dev libsoup2.4-dev libtagc0-dev libxml2-dev libzeitgeist-2.0-dev valac"
```
Comment (Bat41) [inline .travis.yml]: I think we don't need json-glib-1.0, libsoup-2.4, libaccounts-glib and libgsignon-glib anymore, they were only used by the Last.fm plugin.

Comment (Bat41) [reply .travis.yml]: And it would be nice to remove these packages from the README too.

**Comment by danrabbit** [comment]
I don’t think it’s a good idea to wholesale remove working features. Libgpod can probably go since it seems unmaintained, but there are still people who have CD drives or want scrobbling

**Pull request closed without merging**